ui/src/components/Notify/UserMessage.module.css

```diff
@@ -0,0 +1,42 @@
+.messageBody {
+  position: flex;
+  justify-content: flex-end;
+  flex-direction: column-reverse;
+  background-color: rgba(255, 255, 255, 0);
+  display: 'flex';
+  z-index: 1000;
+}
+
+.message {
+  position: relative;
+  padding: 0.4em;
+  border-color: #333;
+  opacity: 1;
+}
+
+.INFO {
```

Comment:
I would use a `data-` attribute for this - e.g. `data-level="INFO"` and then:

```css
.message[data-level="INFO"] {...}
```

It should simplify the JS logic as well.